grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -1,16 +1,14 @@
 package au.org.emii.portal
 
-import grails.gsp.PageRenderer
+import groovy.xml.StreamingMarkupBuilder
```

Comment:
I removed a bunch of unused imports when I had to edit this file.

There was also a fully-qualified class name here https://github.com/aodn/aodn-portal/pull/2056/files#diff-cf1774ac4be79e524ea06d4aaffd87efL47 which seemed a bit odd as you generally only do that (IMO) if you have naming conflicts within a file (e.g. two classes called JSON but from different packages), so I moved it to an import statement.